Automattic/wp-calypso
# Store Promotions: "Applies To" products/categories

This adds support for a promotion to apply to any number of products, or
any number of categories.

To Test:
1. `npm start`
2. Visit `http://calypso.localhost:3000/store/promotion/<site url>`
3. Select "Coupon"
4. Select Applies to "Products"
5. Click the checkboxes on several products.
6. Select Applies to "Product Categories"
7. Click checkboxes on categories.
8. Select Applies to "Products"
9. Observe that the previously checked boxes have been cleared.

(Note: only All products is supported in this PR.)
(This also will require further styling for the checkboxes as well, but functionality is working in this PR)

![new_promotion_ _coderkevin_ _wordpress_com](https://user-images.githubusercontent.com/945228/31480005-d945bea4-aee0-11e7-9e15-ac4350b9910b.png)
